index.md

```diff
@@ -14,31 +14,20 @@ Note: the nested block doesn't doesn't include function or class definition
 as a block, so the example below is considered to have *four* nested blocks,
 not five.
 
-```python
-def my_f(num):
-    if num > 0:
-        if num > 3:
-            if num > 10:
-                if num < 50:
-                    print(num)
-    else:
-        print('done')
-```
```

Comment:
This is a good point. I'll be setting up a live version of the current website off of my personal website in the next few days. (Of course, you won't be able to see pull request changes there, but those should be manually tested anyway.)